cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,52 @@
-describe('', () => {
-  before(() => {
+describe('Article flow', () => {
+  let user;
+  let articleData;
 
+  beforeEach(() => {
+    cy.visit('');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      articleData = generateArticle;
+    });
```

Comment:
Consider moving the generation of the user and article data to a `before` hook instead of `beforeEach`. Since the user and article data do not seem to change across tests, it would be more efficient to generate them only once before all tests run.